src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssStyle = {};
+
+  sourceString
+    .split(';')
+    .map((style) => style.trim())
+    .filter((style) => style.length)
+    .map((style) => {
+      const [key, value] = style.split(':').map((part) => part.trim());
+
+      cssStyle[key] = value;
+    });
```

Comment:
[CODE KNOWLEDGE] - Since you're using the map method to iterate and directly modify the cssStyle object, this is not the best approach. The map method is intended to return a new array and should not be used for side effects. Consider using forEach or reduce to accumulate your results into the cssStyle object.